src/main/java/meditracker/medication/MedicationManager.java

```diff
@@ -72,6 +77,30 @@ private static void checkForDuplicateMedication(String name) throws MediTrackerE
         }
     }
 
+    /**
+     * Checks if the user has input a valid expiry date format.
+     * Also checks that the expiry date entered is not expired.
+     *
+     * @param expiryDate Expiry Date in yyyy-MM-dd format
+     * @throws DateTimeParseException When the date entered is in the wrong format
+     * @throws MediTrackerException When the date entered is already expired
+     */
+    public static void checkforValidExpiryDate(String expiryDate)
+            throws DateTimeParseException, MediTrackerException {
```

Comment:
`DateTimeParseException` is not being thrown by this method but rather caught and handled with, do remove it
Likewise, remove it from the `addMedication` method too

Reply:
Implemented in [41b4727](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/220/commits/41b4727d6e9034177cc15b2e64c72417d26cf55e)